unit/mr_test.c

```diff
@@ -48,58 +48,80 @@ static char err_buf[512];
  */
 static int mr_reg()
 {
-	int i;
+	int i, j;
 	int ret = 0;
-	int testret;
+	int testret = FAIL;
 	struct fid_mr *mr;
+	uint64_t access;
+	uint64_t *access_combinations;
 
-	testret = FAIL;
+	access = ft_info_to_mr_access(fi);
+	ft_alloc_bit_combo(0, access, &access_combinations);
```

Comment:
We want to return the size of the allocated array here, and add a check for success.